lib/svg-sprite/shape.js

```diff
@@ -20,9 +20,7 @@ xpath						= require('xpath'),
 cssom						= require('cssom'),
 CssSelectorParser			= require('css-selector-parser').CssSelectorParser,
 csssel						= new CssSelectorParser(),
-execFile					= require('child_process').execFile,
-phantomjs					= require('phantomjs-prebuilt').path,
-dimensionsPhantomScript		= path.resolve(__dirname, 'shape/dimensions.phantom.js'),
+puppeteer				= require('puppeteer'),
```

Comment:
This needs an additional indent
```suggestion
puppeteer					= require('puppeteer'),
```

Reply:
fixed